app/controllers/passwordless/sessions_controller.rb

```diff
@@ -180,6 +190,13 @@ def find_authenticatable
       end
     end
 
+    def mock_authenticatable
+      email = passwordless_session_params[email_field].downcase.strip
+      resource = authenticatable_class.new(email: email)
+      resource.instance_variable_set(:@mocked, true)
```

Comment:
While this is a clever solution I also think it's a little bit too invasive, I guess. I wonder if we can come up with another solution that completely avoids having to mock anything?